packages/firebase/index.d.ts

```diff
@@ -4238,12 +4265,30 @@ declare namespace firebase.analytics {
      * the gtag.js reference documentation}.
      */
     logEvent(
-      eventName: 'select_content',
+      eventName: 'select_item',
       eventParams: {
         items?: EventParams['items'];
-        promotions?: EventParams['promotions'];
-        content_type?: EventParams['content_type'];
-        content_id?: EventParams['content_id'];
+        item_list_name?: EventParams['item_list_name'];
+        item_list_id?: EventParams['item_list_id'];
+        [key: string]: any;
+      },
+      options?: firebase.analytics.AnalyticsCallOptions
+    ): void;
+
+    /**
+     * Sends analytics event with given `eventParams`. This method
+     * automatically associates this logged event with this Firebase web
+     * app instance on this device.
+     * List of official event parameters can be found in
```

Comment:
Same comment as above RE "official" and rephrasing.

Reply:
Same as above, I've also seen "default".  We definitely don't want users to think they can't use any other strings.

Reply:
Yup, makes sense.  If you think "default" is better than "recommended," let's go with that, thanks!

Reply:
Changed to "recommended".